src/libraries/Common/tests/System/Security/Cryptography/AlgorithmImplementations/TripleDES/TripleDESCipherTests.cs

```diff
@@ -40,6 +41,392 @@ public static void TripleDESInvalidKeySizes()
+            TestTripleDESTransformDirectKey(
+                CipherMode.CFB,
+                PaddingMode.None,
+                key: "04d923abd9291c3e4954a8b52fdabcc804d923abd9291c3e".HexToByteArray(),
+                iv: "191f8794944e601c".HexToByteArray(),
+                plainBytes: "6fe8f67d2af1".HexToByteArray(),
+                cipherBytes: "3bd78a8d24ad".HexToByteArray(),
+                feedbackSize: 8
+            );
+        }
+
+        [Fact]
+        public static void VerifyKnownTransform_CFB8_NoPadding_6()
+        {
+            // NIST CAVS TDESMMT.ZIP TCFB8MMT2.rsp, [DECRYPT] COUNT=6
+            TestTripleDESTransformDirectKey(
+                CipherMode.CFB,
+                PaddingMode.None,
+                key: "a7799e7f5dfe54ce13376401e96de075a7799e7f5dfe54ce".HexToByteArray(),
+                iv: "370184c749d04a20".HexToByteArray(),
+                plainBytes: "2b4228b769795b".HexToByteArray(),
+                cipherBytes: "6f32e4495e4259".HexToByteArray(),
+                feedbackSize: 8
+            );
+        }
+
+        [Fact]
+        public static void VerifyKnownTransform_CFB8_NoPadding_7()
+        {
+            // NIST CAVS TDESMMT.ZIP TCFB8MMT2.rsp, [DECRYPT] COUNT=7
+            TestTripleDESTransformDirectKey(
+                CipherMode.CFB,
+                PaddingMode.None,
+                key: "6bfe3d3df8c1e0d34ffe0dbf854c940e6bfe3d3df8c1e0d3".HexToByteArray(),
+                iv: "51e4c5c29e858da6".HexToByteArray(),
+                plainBytes: "4cb3554fd0b9ec82".HexToByteArray(),
+                cipherBytes: "72e1738d80d285e2".HexToByteArray(),
+                feedbackSize: 8
+            );
+        }
+
+        [Fact]
+        public static void VerifyKnownTransform_CFB8_NoPadding_8()
+        {
+            // NIST CAVS TDESMMT.ZIP TCFB8MMT2.rsp, [DECRYPT] COUNT=8
+            TestTripleDESTransformDirectKey(
+                CipherMode.CFB,
+                PaddingMode.None,
+                key: "e0264aec13e63db991f8c120c4b9b6dae0264aec13e63db9".HexToByteArray(),
+                iv: "bd8795dba79930d6".HexToByteArray(),
+                plainBytes: "79068e2943f02914af".HexToByteArray(),
+                cipherBytes: "9b78c5636c5965f88e".HexToByteArray(),
+                feedbackSize: 8
+            );
+        }
+
+        [Fact]
+        public static void VerifyKnownTransform_CFB8_NoPadding_9()
+        {
+            // NIST CAVS TDESMMT.ZIP TCFB8MMT2.rsp, [DECRYPT] COUNT=9
+            TestTripleDESTransformDirectKey(
+                CipherMode.CFB,
+                PaddingMode.None,
+                key: "7ca28938ba6bec1ffec78f7cd69761947ca28938ba6bec1f".HexToByteArray(),
+                iv: "953896586e49d38f".HexToByteArray(),
+                plainBytes: "2ea956d4a211db6859b7".HexToByteArray(),
+                cipherBytes: "f20e536674a66fa73805".HexToByteArray(),
+                feedbackSize: 8
+            );
+        }
+
+
+
+        [Fact]
```

Comment:
One blank line would be sufficient :smile:.

```suggestion
        }

        [Fact]
```